app/workers/commit_monitor_handlers/commit/bugzilla_checker.rb

```diff
@@ -0,0 +1,112 @@
+module CommitMonitorHandlers
```

Comment:
it's hard to see what changed here! it looks like in `#process_commit` the worker will either do one thing or the other. what do you think about having a separate worker for each instead? then maybe any code they both need could be extracted into something they could both share. not sure about the feasibility of this....any thoughts?